atomic-design/atoms/button/stories/button.stories.tsx

```diff
@@ -41,25 +41,46 @@ Simple.args = {
 
 const PseudoStatesTemplate: Story<ButtonProps> = ({ children, ...args }) => {
 	return (
-		<div style={{ width: "max-content" }}>
-			<Button {...args} type="button">
-				{children}
-			</Button>
-		</div>
+		<Button {...args} type="button">
+			{children}
+		</Button>
 	);
 };
 export const PseudoStates = PseudoStatesTemplate.bind({});
```

Comment:
Shouldn't this not `.bind(null, {})`, because the first argument of `bind` is `thisArg`?

Reply:
I think this might be unneeded in general. I'll see if we can optimize it. I generally want to remove all controls (and maybe even the tab for them if possible).